### ISSUE TITLE
Can't have the same results as Mura team

### ISSUE THREAD

**Issue by lioGuillaume** [pyaf/DenseNet-MURA-PyTorch#9]
Hi Pyaf, 

Thanks for your work. 
Did you have the same results as the Mura team (kappa = 0.931 on the wrist) ?
I can't reach it.

**Comment by fahadakhan96** [comment]
Yes! I'd like to know about this, too. I tried it on my end and the best accuracy I got was 0.6582.
